comit/src/btsieve/ethereum/transaction_pattern.rs

```diff
@@ -172,42 +172,12 @@ pub struct Event {
 #[cfg(test)]
 mod tests {
     use super::*;
-    use crate::{
-        ethereum::{
-            Address, Block, Bytes, Log, Transaction, TransactionReceipt, H160, H2048, H256,
-        },
-        quickcheck::Quickcheck,
+    use crate::ethereum::{
+        Address, Block, Bytes, Log, Transaction, TransactionReceipt, H160, H2048, H256,
     };
     use spectral::prelude::*;
     use std::str::FromStr;
 
-    #[test]
-    fn given_pattern_from_arbitrary_address_contract_creation_transaction_matches() {
```

Comment:
❓ Why was this test removed? Was it re-introduced somewhere or is it just not necessary...?

Reply:
:D

I could not be bothered duplicating the quickcheck stuff in two crates so I just removed this test because I don't think it actually adds value.